.travis.yml

```diff
@@ -23,26 +18,38 @@ jobs:
   - python: 3.8-dev
     dist: xenial
     sudo: true
-    stage: extra
   - python: nightly
     dist: xenial
     sudo: true
-    stage: extra
   - python: pypy   # 2.7.13 pip  9.0.1
-    stage: extra
   - python: pypy3  # 3.5.3  pip  9.0.1
-    stage: extra
   - os: osx
     language: generic
-    stage: extra
   - python: 3.7
     dist: xenial
     sudo: true
     env: TOXENV=docs
-    stage: quick
   - python: 3.6
-    stage: quick
     env: TOXENV=fix-lint
+  - stage: deploy
+    python: 3.7
+    dist: xenial
+    sudo: true
+    install: pip install -U "pip >= 10.0.0"
+    script: skip
+    after_success: skip
+    env: TOXENV=release
+    deploy:
+      - provider: pypi
+        user: toxdev
+        distributions: sdist bdist_wheel
+        skip_upload_docs: true
+        password:
+          secure: GpPLWTVHMAOpHh2Jq+cYo9hE6cwTxPJ6fMA+q+ERWHtUAqSC88kBqoZ4DQb0cyFIlN24Z9f5J2vm80Q7Qv2UWCGVKeNaWMWFqvZ+P5czGEn7ARoeTsTaiyKTCC2w/7iDsne7dPcSHT/y8V2fO/35kcwcrc0EGr4peb9GDJ4asTFpffiVzW9l5OjV2+jQn+ihWNA0ujSB3AR4TLexXGH+Gf6O5uEHK/kPJ63NEDbJVEdciWqLbDiU+OwWl5Xb5hFX1fjKQ9dP4zA7SolNsNB6pqs4rLl2aHqKYOt1RYGgjWSvANDXKP7S46Q5LFIPkmt4QbOeWxX63NUMZUmWadbjoAF65q1d9Dq41IemsLmElSqpdfPaLtLgLCbNlgHRLNVH+nJVqxjc3bvdxi28K8DZ9BB9KGmPkrbbj4NB0prsPouKiBkUS9NyoQuDt1EzJ3Rn2FzMOH41X5gIqzdsdxcy+3DV2Bb+eozNgg+51n27ESf7vwiYMW9GFENw/6AcPkxIPYakQu4joJ+327wVfK4t0gdy/m4qu2n0Lx6wOJUdSOcOAT5bPaRBQV92gXAaVUsROKMfrzwlVLuA+GXcnj8fft8aMXem30+N3r7y5FjbiAts345qnbjSWV3w9qEbNjM/Z9mt7z1OfsqCarJ2Mj5wKQyYfewddYMjdk7aPW5/9gQ=
+        on:
+          tags: true
+          type: pull_request
```

Comment:
I'm curious about this option, I don't see it listed in [the docs](https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A)... is it triggered only for PRs on `tox-dev/tox`, as opposed to PRs from forks?

Reply:
Indeed is not there for the build, however, it's there for the general conditionals settings https://docs.travis-ci.com/user/conditions-v1#Integration. 

That being said I did not test it yet, will do on Friday. If it does not work here maybe we can make the entire deploy stage conditional on pull request only, and have the tags only here.

Reply:
OK, thanks!